### PR TITLE
[CodeCamp2023-474] Add new configuration files for DINO algorithm in mmdetection. 

### PR DESCRIPTION
**Pull request by Markson-Young**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Add new configuration files for DINO algorithm in mmdetection.

## Modification

new config file for DINO algorithm in mmdet dev-3.x.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMPreTrain.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdet/configs/dino/dino_5scale_swin_l_8xb2_12e_coco.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmengine.config import read_base
+from mmengine.model.weight_init import PretrainedInit
+
+from mmdet.models import SwinTransformer
+
+with read_base():
+    from .dino_4scale_r50_8xb2_12e_coco import *
+
+fp16 = dict(loss_scale=512.)
```
Comment (hhaAndroid) [inline mmdet/configs/dino/dino_5scale_swin_l_8xb2_12e_coco.py:10]: 这个是不需要的，可以删掉

Comment (hhaAndroid) [reply mmdet/configs/dino/dino_5scale_swin_l_8xb2_12e_coco.py:10]: 麻烦把旧config 里面的对应字段也删掉吧

Comment (Markson-Young) [reply mmdet/configs/dino/dino_5scale_swin_l_8xb2_12e_coco.py:10]: 请问是要删除第10行：fp16 = dict(loss_scale=512.)这一句代码吗？向你确认一下

Comment (hhaAndroid) [reply mmdet/configs/dino/dino_5scale_swin_l_8xb2_12e_coco.py:10]: 是的，这个其实是旧的无效配置

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**